.github/actions/setup/action.yaml

```diff
@@ -5,9 +5,8 @@ inputs:
     description: Set up Rust nightly toolchain?
     required: false
   target:
-    description: Rust toolchain to use
-    required: false
-    default: ""
+    description: Rust target triple to use
```

Comment:
is "target triple" a typo?

Reply:
I don't understand the intent of your question. I think this is an official term.
<https://doc.rust-lang.org/beta/rustc/platform-support.html#platform-support>

Reply:
My bad, I just didn't know this expression